Project.toml

```diff
@@ -22,14 +22,17 @@ ValSplit = "0625e100-946b-11ec-09cd-6328dd093154"
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
 GraphMakie = "1ecd5474-83a3-4783-bb4f-06765db800d2"
+GraphRecipes = "bd48cda9-67a9-57be-86fa-5b3c104eda73"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
I think Plots is not used directly in the package extension. Do you mind trying if it works without this weak dep? This way we can avoid compat constraints.